src/java/org/apache/cassandra/sensors/RequestSensors.java

```diff
@@ -46,6 +48,8 @@ public class RequestSensors
 {
     private final Supplier<SensorsRegistry> sensorsRegistry;
     private final ConcurrentMap<Pair<Context, Type>, Sensor> sensors = new ConcurrentHashMap<>();
+    private final ConcurrentMap<Sensor, Double> latestSyncedValuePerSensor = new ConcurrentHashMap<>();
```

Comment:
This is just for my understanding, not a change request: I guess this could be a non-concurrent map as it's guarded by the write lock? Is there anything I am missing that requires it to be concurrent? As said, totally fine with a concurrent map, I just want to understand if I am missing any concurrency issues here.

Reply:
Right this doesn't have to be a CHM, it's a leftover from my previous implementation where I tried to commit suicide with a lock-free implementation before surrendering to the Gods of Concurrency :)